deploy/lib/xquery/setup.xqy

```diff
@@ -1131,75 +1396,160 @@ declare function setup:validate-forests-from-config(
   let $data-directory as xs:string? := $forest-config/as:data-directory[fn:string-length(fn:string(.)) > 0]
   let $host-name as xs:string? := $forest-config/as:host-name[fn:string-length(fn:string(.)) > 0]
   let $replica-names as xs:string* := $forest-config/as:replica-names/as:replica-name[fn:string-length(fn:string(.)) > 0]
-  return
+  let $a :=
```

Comment:
`$a` doesn't do anything to improve the code's self-documentation. Restore to returning the result of `setup:validate-forest` directly.